dotcms-integration/src/test/java/com/dotcms/MainSuite2b.java

```diff
@@ -79,6 +79,7 @@
 import com.dotcms.rest.api.v1.asset.AssetPathResolverImplIntegrationTest;
 import com.dotcms.rest.api.v1.asset.WebAssetHelperIntegrationTest;
 import com.dotcms.rest.api.v1.authentication.ResetPasswordTokenUtilTest;
+import com.dotcms.rest.api.v1.job.JobQueueHelperIntegrationTest;
```

Comment:
@fabrizzio-dotCMS I just see the import but I don't see it added inside the `@SuiteClasses({`